site/source/user-manual/service-guides/lightning.rst

```diff
@@ -8,10 +8,163 @@ Lightning
+    :width: 60%
+
+   Enter the copied password and log in:
+
+   .. figure:: /_static/images/lightning/rtl-enter-pass.png
+    :width: 60%
+
+#. Once in RTL, click **"On-chain"** then click **"Generate Address"**
+
+   .. figure:: /_static/images/lightning/rtl-generate-address.png
+    :width: 60%
+
+#. Send money to the generated address to add funds to your lightning wallet:
+
+   .. figure:: /_static/images/lightning/rtl-address-generated.png
+    :width: 60%
+
+   .. note:: Please do not send money to the address pictured above as we will not receive it. If you are intent on sending us money please `head here <donate.start9.com>`_
+
+#. Once your sats confirm on-chain you'll see this:
+
+   .. figure:: /_static/images/lightning/rtl-on-chain.png
+    :width: 60%
+
+#. Now we must add a peer with which to make channels. In this example we will be opening a channel with Start9 so we will add Start9's node as a peer. Click **"Peers/Channels"**:
+
+   .. figure:: /_static/images/lightning/rtl-peers-channels.png
+    :width: 60%
+
+#. Click **"Peers"** then **"Add Peer"**:
+
+   .. figure:: /_static/images/lightning/rtl-peers-add-peer.png
+    :width: 60%
+
+#. Enter **025d28dc4c4f5ce4194c31c3109129cd741fafc1ff2f6ea53f97de2f58877b2295@64.225.19.231:9735** - these are the peer details for Start9's lightning node - and click **"Add Peer"**:
+
+   .. figure:: /_static/images/lightning/rtl-start9-node-info.png
+    :width: 60%
+
+#. Then you can enter an amount (the size of the channel), select Private Channel (unless you want a public channel - see below), and a Fee Rate (check a `block explorer <https://mempool.space>`_ for an idea of current necessary fees):
+
+   .. figure:: /_static/images/lightning/rtl-open-channel.png
+    :width: 60%
+
+   .. note:: Using a private channel is what we advise as a default. You may wish for the channel to be public if you intend on becoming a routing node or for other reasons.
+
+#. You will now see your channel in **"Channels"** -> **"Pending/Inactive"**:
+
+   .. figure:: /_static/images/lightning/rtl-pending-inactive.png
+    :width: 60%
+
+#. Once the transaction opening the channel gets added to a block your channel will soon appear here under **"Open"**:
+
+   .. figure:: /_static/images/lightning/rtl-open-channels.png
+    :width: 60%
+
+#. To make a payment head to the **"Transactions"** tab and press **"Send Payment"**:
+
+   .. figure:: /_static/images/lightning/rtl-transactions-tab.png
+     :width: 60%
+
+#. That's it! You now have a lightning node running with a channel open ready to send payments on the lightning network!
+
+.. note:: You will not be able to receive payments until you have inbound liquidity in your channel. After completing the above process you will only have outbound liquidity. Inbound liquidity can be created by making payments, having someone open a channel to you or via more sophisticated channel creation.
+
+
+Lightning Wallets
```

Comment:
Let's rename to "Lightning Integrations," or "Lightning Interfaces" as wallet is a misnomer